slp-postage-protocol.md

```diff
@@ -0,0 +1,268 @@
+      {
+         "name":"Spice Token",
+         "symbol":"SPICE",
+         "tokenId":"4de69e374a8ed21cbddd47f2338cc0f479dc58daa2bbe11cd604ca488eca0ddf",
+         "decimals":8,
+         "rate":727418066
+      },
+      {
+         "name":"Honest Coin",
+         "symbol":"USDH",
+         "tokenId":"c4b0d62156b3fa5c8f3436079b5394f7edc1bef5dc1cd2f9d0c4d46f82cca479",
+         "decimals":2,
+         "rate":1
+      },
+      {
+         "name":"AnyPay Gold",
+         "symbol":"GOLD",
+         "tokenId":"8e635bcd1b97ad565b2fdf6b642e760762a386fe4df9e4961f2c13629221914f",
+         "decimals":3,
+         "rate":1
+      }
+   ]
+}
+```
+
+#### Integration With Simple Ledger Payment Protocol
+
+A merchant utilizing [Simple Ledger Payment Protocol](https://github.com/simpleledger/slp-specifications/blob/master/slp-payment-protocol.md) to process payments can, optionally, act as a post office for buyers. In such a case, the Payment may be sent to the payment protocol server with an additional SLP output paying for postage. The merchant's server will add the required satoshis, after validating the payment and before broadcasting the transaction to the Bitcoin Cash network. A merchant wishing to offer this service will include the Postage Rate object in the Payment Request sent to the buyer. The Postage Rate object will be contained in the `postage` property of the [JSON Merchant Data](https://github.com/jeton-tech/payment-protocol-extensions/blob/master/json-merchant-data.md) of the Payment Request.
+
+#### Payment Request JSON Merchant Data Object Example
+```
+{
+   "postage":
+      {
+         "version":1,
+         "address":"simpleledger:qrxj0mftnsrl63uqwn2jcsxvwymgxm7sev7dyx7hrr",
+         "weight":365,
+         "transactionttl":30,
+         "stamps":[
+            {
+               "name":"Spice Token",
+               "symbol":"SPICE",
+               "tokenId":"4de69e374a8ed21cbddd47f2338cc0f479dc58daa2bbe11cd604ca488eca0ddf",
+               "decimals":8,
+               "rate":727418066
+            },
+            {
+               "name":"Honest Coin",
+               "symbol":"USDH",
+               "tokenId":"c4b0d62156b3fa5c8f3436079b5394f7edc1bef5dc1cd2f9d0c4d46f82cca479",
+               "decimals":2,
+               "rate":1
+            },
+            {
+               "name":"AnyPay Gold",
+               "symbol":"GOLD",
+               "tokenId":"8e635bcd1b97ad565b2fdf6b642e760762a386fe4df9e4961f2c13629221914f",
+               "decimals":3,
+               "rate":1
+            }
+         ]
+      }
+}
+```
+
+
+### Payment Generation
+
+Payments are generated according to the Payment Generation section of the [Simple Ledger Payment Protocol](https://github.com/simpleledger/slp-specifications/blob/master/slp-payment-protocol.md) with the following additional considerations:
+
+1. Only valid Simple Ledger Protocol inputs and outputs may be included. No non-SLP inputs or outputs may be included in the transaction
```

Comment:
This is a redundant comment :) Is there a reason why not to allow BCH as a form of stamp payment?